packages/lib-classifier/src/components/Classifier/Classifier.js

```diff
@@ -94,6 +95,11 @@ export default function Classifier({
   })
 
   const [loaded, setLoaded] = useState(false)
+  const { data } = useSWR(`/workflows/${workflowID}`, client.panoptes.get)
```

Comment:
We could add an access token to this request, so that project owners can preview private workflows in the new classifier.